benchmarks/src/main.rs

```diff
@@ -27,6 +27,9 @@ fn init() {
                 .collect(),
         );
     });
+
+    // Set a current time to avoid header validation failures due to blocks appearing to be in the future
+    ic_btc_canister::runtime::mock_time::set_mock_time_secs(1750077112); // June 2025
```

Comment:
NIT: maybe put this magic number into a constant with corresponding comment

```rs
// Set a fixed mock time (approx. June 2025) to avoid "block from the future" errors.
// Calculated as: 55.5 years * 365.25 days/year * 24h * 60m * 60s ≈ 1_750_000_000 seconds since 1970-01-01 UTC.
let june_2025 = (55.5 * 365.25 * 24.0 * 60.0 * 60.0) as u64;
ic_btc_canister::runtime::mock_time::set_mock_time_secs(june_2025);
```